### DISCUSSION TITLE
FastAPI + MongoDB Change Stream + Websockets

### DISCUSSION THREAD

**Discussion by rozeappletree** [fastapi/fastapi#8560] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
None
```

### Description

I am looking for a standalone code implementation like this https://www.mongodb.com/developer/how-to/subscribing-changes-browser-websockets/ where I can use FastAPI along with mongodb change streams as well as websockets. 

### Operating System

Linux

### FastAPI Version

Latest

### Python Version

3.7+

**Comment by rozeappletree** [comment]
I had a look at how connection manager was implemented in FastAPI websockets tutorial as a GLOBAL VARIABLE. It is not a good practice right when we scale the API, global variable is not going to stay same in multiple instances

**Reply by YuriiMotov** [reply]
> I had a look at how connection manager was implemented in FastAPI websockets tutorial as a GLOBAL VARIABLE. It is not a good practice right when we scale the API, global variable is not going to stay same in multiple instances

There is a remark about this in current version:

> But keep in mind that, as everything is handled in memory, in a single list, it will only work while the process is running, and will only work with a single process.

https://fastapi.tiangolo.com/advanced/websockets/#more-info
